packages/wp-source/src/libraries/get-match.ts

```diff
@@ -56,10 +56,30 @@ interface GetMatchReturn<
   pattern: string;
 }
 
+/**
+ * The `source` object that contains a link and a route.
+ */
+interface Source {
+  /**
+   * The full link with `queryString`.
+   */
+  link?: string;
+
+  /**
+   * The link without the query part. Also known as the pathname.
```

Comment:
There is need to change the TSDocs, but just to make it clear, the route is not the pathname but the pathname without the pagination: 

- link: `/category/some-category/page/3?a=1`
- pathname: `/category/some-category/page/3`
- route: `/category/some-category/`

We didn't find a better name for that so we used `route` 😆🤷

Reply:
Totally 😅 . We'll keep this in mind on next source version and we'll name it better.

Reply:
> We'll keep this in mind on next source version and we'll name it better.

It's open to suggestions 👍